frontend/src/metabase/components/Popover/Popover.css

```diff
@@ -45,6 +45,7 @@
 }
 
 .tippy-box[data-theme~="popover"] {
+  font-size: inherit;
```

Comment:
Prevent some text that relies on `em` to have weird size due to `.tippy-box`'s `font-size: 14px`

Without this line, the popover would look like this.
![image](https://user-images.githubusercontent.com/1937582/162722201-829ca374-8b90-4495-b70d-2d30283be388.png)